Squeezer/src/main/java/uk/org/ngo/squeezer/NowPlayingFragment.java

```diff
@@ -658,15 +660,22 @@ private void updateSongInfo(@NonNull PlayerState playerState) {
 
             if (mFullHeightLayout) {
                 artistText.setText(song.getArtist());
+                albumText.setText(song.getAlbumName());
                 totalTime.setText(Util.formatElapsedTime(song.getDuration()));
             }
+            else {
+                artistAlbumText.setText(song.getArtist() + " - " + song.getAlbumName());

```

Comment:
As Kurt noted, this will set the text to " - " for things that don't have artists or albums, e.g., some radio streams (I verified this by streaming http://www.hitzradio.com/hitzradio.pls.

A fix would be to:

1. Put the hint back.

2. Change this to

```
                artistAlbumText.setText(mJoiner.join(
                        Strings.emptyToNull(song.getArtist()),
                        Strings.emptyToNull(song.getAlbumName())));
```

mJoiner can be defined in the same way as in `BaseItemView.java`.